Sources/applozickit/ALMQTTConversationService.m

```diff
@@ -307,6 +307,19 @@ - (void)newMessage:(MQTTSession *)session
 
             ALPushAssist *pushAssist = [[ALPushAssist alloc] init];
             ALMessage *alMessage = [[ALMessage alloc] initWithDictonary:[theMessageDict objectForKey:@"message"]];
+            if ([alMessage isAssignmentMessage]) {
+                [ALMessageService getLatestMessageForUser:[ALUserDefaultsHandler getDeviceKeyString] withDelegate:self.realTimeUpdate
```

Comment:
Add a comment why this is needed.

Reply:
added comment & optimized the if block logic so that